requirements.txt

```diff
@@ -17,3 +17,4 @@ dataclasses>=0.6; python_version<"3.7"
 swagger-spec-validator>=2.7.4
 psutil
 jsonschema<4.0.0
+retry<1.0.0
```

Comment:
If there is any important reason for compability, it's worth mentioning here in comment, so we could check if problem exists in the future when we'd like to upgrade requirements.

Reply:
current version of retry is 0.9.2 - https://pypi.org/project/retry/   
I want to avoid situation when they release version 1+ with broken backward compatibility. It is the reason why such requirement was added.